owasp-top10-2017-apps/a5/tictactoe/deployments/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:8.15.1-alpine
```

Comment:
Changing nodes version breaks execution of the app during registration and login.

As debugged with @aranhams it appears in new node versions the promise must dealt with, which would in turn fix the vulnerability.

Reply:
I added some error handling in the functions that return a Promise and luckily the application is still vulnerable 🤣